pkg/schedule/config/config.go

```diff
@@ -314,6 +323,12 @@ type ScheduleConfig struct {
 
 	// PatrolRegionWorkerCount is the number of workers to patrol region.
 	PatrolRegionWorkerCount int `toml:"patrol-region-worker-count" json:"patrol-region-worker-count"`
+
+	// EnableAffinityScheduling is the option to enable affinity scheduling.
+	EnableAffinityScheduling bool `toml:"enable-affinity-scheduling" json:"enable-affinity-scheduling,string,omitempty"`
+	// If the size of region is smaller than MaxAffinityMergeRegionSize,
```

Comment:
Should the affinity's merge consider the key count?

Reply:
Yes, it will calculate by MaxAffinityMergeRegionSize 
<img width="892" height="248" alt="image" src="https://github.com/user-attachments/assets/466f7d11-c129-4219-ad09-bf995893f71d" />

Reply:
done